src/app/components/Headings/index.jsx

```diff
@@ -28,7 +28,16 @@ export const Headline = styled.h1`
   }
 `;
 
-export const SubHeading = styled.h2`
+const regexRemoveUnnecessarySymbols = /[.,/#!$%^&*;:{}=\-_~()'"]/g;
+const regexRemoveSpacingReplaceWithDash = /\s+/g;
```

Comment:
Technically, this regex just matches the spaces. It's the replace method below that does the replacing. Renaming this to `regexSpaces` and the one above it as `regexPunctuationSymbols` would be more accurate.